app/forms/ip-pool-create.tsx

```diff
@@ -51,6 +52,10 @@ export function CreateIpPoolSideModalForm() {
     >
       <NameField name="name" control={form.control} />
       <DescriptionField name="description" control={form.control} />
+      <Message
+        variant="info"
+        content="IP pool names and descriptions are visible to end users in linked silos."
+      />
```

Comment:
I like this — since we have so much space and the form is so simple, we phave room to add a few more words about why or when users will see this. Also I think we can just call them "users" instead of end users. Something like this?

> IP pool names and descriptions are shown to users in linked silos in order to help them choose a pool while allocating IPs for instances.